ee/clickhouse/views/groups.py

```diff
@@ -56,4 +57,4 @@ def property_values(self, request: request.Request, **kw):
             {"team_id": self.team.pk, "group_type_index": request.GET["group_type_index"], "key": request.GET["key"]},
         )
 
-        return response.Response([{"name": name} for name in rows])
+        return response.Response([{"name": convert_property_value(name[0])} for name in values])
```

Comment:
So this isn't the correct fix:

`sync_execute` returns a list of rows - i.e. a list of lists, example: `[['techology'], ['finance']]`

The list comprehension here is wrong because iterating over the rows it doesn't yield a name, but a list containing the name.

`convert_property_value` does nothing relevant for us here, not sure why it was added in the first place. Taking a look at the implementation: `name` won't be a boolean or a dict or a list but always a string. While it's being used in other places, that doesn't automatically quality it here. 

I recommend to always e.g. print out the returning data structure and work from there.

You want this here:
```python
return response.Response([{"name": row[0]} for row in rows])
```